cluster/common.sh

```diff
@@ -603,6 +603,7 @@ KUBELET_TOKEN: $(yaml-quote ${KUBELET_TOKEN:-})
 KUBE_PROXY_TOKEN: $(yaml-quote ${KUBE_PROXY_TOKEN:-})
 ADMISSION_CONTROL: $(yaml-quote ${ADMISSION_CONTROL:-})
 MASTER_IP_RANGE: $(yaml-quote ${MASTER_IP_RANGE})
+MASTER_RESERVED_IP: $(yaml-quote ${MASTER_RESERVED_IP})
```

Comment:
From https://cloud.google.com/compute/pricing: "... there are charges for egress through external IP addresses, regardless of whether the instances are in the same zone." so using an external IP would increase pricing.